Added date to release page (#204)
adoptium/adoptium.net: content/blog/eclipse-temurin-8u462-11028-17016-2108-2402-available/index.md

## Fixes and Updates

This release contains the following fixes and updates.

* [Temurin 8u462 release notes](https://adoptium.net/temurin/release-notes/?version=jdk8u462-b08)

* [Temurin 11.0.28 release notes](https://adoptium.net/temurin/release-notes/?version=jdk-11.0.28+6)

* [Temurin 17.0.16 release notes](https://adoptium.net/temurin/release-notes/?version=jdk-17.0.16+8)

* [Temurin 21.0.8 release notes](https://adoptium.net/temurin/release-notes/?version=jdk-21.0.8+9)

* [Temurin 24.0.2 release notes](https://adoptium.net/temurin/release-notes/?version=jdk-24.0.2+12)

## New and Noteworthy

> 

Reviewer: We should call out https://github.com/adoptium/containers/pull/782

Specifically that we've introduced an Alpine 3.22 image which will now be the default Alpine version when someone pulls a tag such as `eclipse-temurin:21-alpine`. They will be able to stick to the previous version if they want by updating their Dockerfiles to pull from `eclipse-temurin:21-alpine-3.21`

Reviewer 2: added

## Fixes and Updates

This release contains the following fixes and updates.

* [Temurin 8u462 release notes](https://adoptium.net/temurin/release-notes/?version=jdk8u462-b08)

* [Temurin 11.0.28 release notes](https://adoptium.net/temurin/release-notes/?version=jdk-11.0.28+6)

* [Temurin 17.0.16 release notes](https://adoptium.net/temurin/release-notes/?version=jdk-17.0.16+8)

* [Temurin 21.0.8 release notes](https://adoptium.net/temurin/release-notes/?version=jdk-21.0.8+9)

* [Temurin 24.0.2 release notes](https://adoptium.net/temurin/release-notes/?version=jdk-24.0.2+12)

## New and Noteworthy

> ### Alpine default image now version 3.22

Reviewer: ```suggestion
### Alpine default container image now version 3.22
```

> Alpine 3.22 is now the default image when pulling eclipse-temurin Docker images. Previous Alpine versions are still available by specifying a suffix of "-3.21" in Dockerfiles, eg.eclipse-temurin:21-alpine-3.21

Reviewer: ```suggestion
Alpine 3.22 is now the default Alpine image when pulling `eclipse-temurin` container images. Previous Alpine versions are still available by specifying a suffix of "-3.21" in Dockerfiles, eg. `eclipse-temurin:21-alpine-3.21`
```

> Alpine 3.22 is now the default image when pulling eclipse-temurin Docker images. Previous Alpine versions are still available by specifying a suffix of "-3.21" in Dockerfiles, eg.eclipse-temurin:21-alpine-3.21

Reviewer: ```suggestion
Alpine 3.22 is now the default image when pulling eclipse-temurin Docker images. Previous Alpine versions are still available by specifying a suffix of "-3.21" in Dockerfiles, eg.`eclipse-temurin:21-alpine-3.21`.

> Alpine 3.22 is now the default image when pulling eclipse-temurin Docker images. Previous Alpine versions are still available by specifying a suffix of "-3.21" in Dockerfiles, eg.`eclipse-temurin:21-alpine-3.21`.

Reviewer 3: ```suggestion
Alpine 3.22 is now the default image when pulling eclipse-temurin Docker images. Previous Alpine versions are still available by specifying a suffix of "-3.21" in Dockerfiles, e.g. `eclipse-temurin:21-alpine-3.21`.
```
